test/Microsoft.Azure.ServiceBus.UnitTests/Microsoft.Azure.ServiceBus.UnitTests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft.Azure.ServiceBus.UnitTests</AssemblyTitle>
     <TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Hey @SeanFeldman - I definitely agree with you, but I have one hesistation:

If someone were to create a 'spiked pr' as you call it, they would then have build errors if they didn't document all public memebers, which would prevent the CI from running. If they wanted to get rid of those warning they would need to have empty tags, at which point those errors wouldn't show up anymore.

Personally, I'd rather see the annoying warning, but continue the CI, and add something to our PR template, saying that there should not be any warnings during compilation.